### PR TITLE
Add a new doc for remote debugging on Azure.

### PR DESCRIPTION
**Pull request by fiveisprime**
We'll be shipping an updated version of the extension next week that
removes the feature flag for remote debugging. This doc supports that
release.

Is there an additional change required to get this in the ToC? If so, that
should be delayed until the end of next week.

### CODE REVIEW

**Comment by fiveisprime** [comment]
@auchenberg, can you review this, please? <3

File: docs/azure/remote-debugging.md [hunk 1]
```diff
@@ -8,3 +8,39 @@ MetaDescription: Azure Remote Debugging with Visual Studio Code
 DateApproved:
 ---
 # Azure Remote Debugging
+
+Connect the Visual Studio Code debugger to your apps running on **Azure App Service** on **Linux**. The debugger works the same as when it's connected to a local Node.js process - including the use of Breakpoints and Logpoints.
```
Comment (auchenberg) [inline docs/azure/remote-debugging.md]: Do we want to make this specific to Node.js for now? "Azure Remote Debugging" is quite generic.

File: docs/azure/remote-debugging.md [hunk 2]
```diff
@@ -8,3 +8,39 @@ MetaDescription: Azure Remote Debugging with Visual Studio Code
...
+
+![Start remote debugging](images/remote-debugging/start-remote-debugging.png)
+
+Once restarted, Visual Studio Code will connect to the app's debugging port via an SSH tunnel. It may take a little time to establish the connection. Once connected, Visual Studio Code will switch into debugging mode and work the same as it does when debugging an app locally.
+
+![Remote breakpoint](images/remote-debugging/remote-breakpoint.png)
+
+When you're ready to end your remote debugging session, disconnect from the debugger as you would normally and confirm that you want to restart the app.
+
+## Next steps
```
Comment (auchenberg) [inline docs/azure/remote-debugging.md]: We can add a #LogPoints paragraph/section linking to https://code.visualstudio.com/docs/editor/debugging#_logpoints

File: docs/azure/remote-debugging.md [hunk 3]
```diff
@@ -8,3 +8,39 @@ MetaDescription: Azure Remote Debugging with Visual Studio Code
 DateApproved:
 ---
 # Azure Remote Debugging
+
+Connect the Visual Studio Code debugger to your apps running on **Azure App Service** on **Linux**. The debugger works the same as when it's connected to a local Node.js process - including the use of Breakpoints and Logpoints.
+
+## Install the extension
+
+Remote debugging support for VS Code is provided by the [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-cosmosdb) extension for apps deployed to Azure. To install the Azure App Service extension, open the Extensions view by pressing `kb(workbench.view.extensions)` and search for 'app service' to filter the results. Select the Microsoft **Azure App Service** extension.
```
[the remark this answers is not in the document]
Comment (fiveisprime) [reply docs/azure/remote-debugging.md]: Good looking out, fixed

File: docs/azure/remote-debugging.md (hunk 1 above)
Comment (fiveisprime) [reply docs/azure/remote-debugging.md]: Changed it to "Azure Remote Debugging for Node.js Apps"

**Comment by auchenberg** [comment]
@fiveisprime Wouldn't it make more sense land this in the Node Dev Center if we anyways want to migrate content over? Less debt for us.

**Comment by fiveisprime** [comment]
I'm indifferent. I started here because I can land it by the end of the week when the extension ships. I have no idea how long it might take to land in docs.

**Comment by auchenberg** [comment]
Let's keep it here for now, and migrated once we figure out the docs things.

File: docs/azure/remote-debugging.md [hunk 4]
```diff
@@ -7,4 +7,40 @@ ContentId: 09cb23b6-b1e9-4a29-a934-cbc16fe109c7
...
+
+Once restarted, Visual Studio Code will connect to the app's debugging port via an SSH tunnel. It may take a little time to establish the connection. Once connected, Visual Studio Code will switch into debugging mode and work the same as it does when debugging an app locally.
+
+![Remote breakpoint](images/remote-debugging/remote-breakpoint.png)
+
+When you're ready to end your remote debugging session, disconnect from the debugger as you would normally and confirm that you want to restart the app.
+
+## Next steps
+
+* [Log Points](https://code.visualstudio.com/docs/editor/debugging#_logpoints) - Use Logpoints to log to the console without "breaking" in the debugger.
```
Comment (auchenberg) [inline docs/azure/remote-debugging.md]: Small nit: Logpoints

**Comment by auchenberg** [comment]
LGTM with the smaller nit.

**Comment by fiveisprime** [comment]
Alright, resolved. Thanks for taking a look. :)

@gregvanl, it's safe to publish Thursday or Friday.

**Pull request merged**